x/pocketcore/types/service_test.go

```diff
@@ -2,17 +2,14 @@ package types
 
 import (
 	"encoding/hex"
-	"github.com/pokt-network/pocket-core/codec"
-	types2 "github.com/pokt-network/pocket-core/codec/types"
-	"github.com/pokt-network/pocket-core/crypto"
-	exported2 "github.com/pokt-network/pocket-core/x/apps/exported"
-	"github.com/pokt-network/pocket-core/x/auth"
-	"github.com/pokt-network/pocket-core/x/gov"
```

Comment:
Why do we need this change?
x/gov/module.go importing x/pocketcore/types caused a cyclic import in the x/pocketcore/types package because this file imported x/gov.

It seems we don't need `makeTestCodec()` at all.  I verified all tests under x/pocketcore/types and x/pocketcore/keeper passed.

Reply:
Sounds reasonable to me